CHANGE_LOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 - Add disclaimer note for prechat sample.
 - Uptake [@microsoft/omnichannel-chat-sdk@1.9.3](https://www.npmjs.com/package/@microsoft/omnichannel-chat-sdk/v/1.9.3)
+- Changed authentication token error handling to issue warnings instead of errors.
```

Comment:
I'd put "Changed `AuthenticatedChatConversationRetrievalFailure` error to issue warnings instead of errors"

It'd be easier to look use `AuthenticatedChatConversationRetrievalFailure` as keyword to search if we ever had to understand how this change works